src/tools/pub/cmd/index.md

```diff
@@ -5,14 +5,9 @@ title: "Pub Commands"
 description: "Pub, a package and asset management tool for Dart, supports a variety of commands."
 ---
 
-Aside from managing packages, [pub](/tools/pub) also has support for
-building web and command line apps.
-
-If your web app's directory structure follows pub's [package layout
-conventions](/tools/pub/package-layout), you can use the pub
-development server (<code class="literal">pub serve</code>) to continuously
-build and serve the app's assets. Once you're ready to deploy the web app,
-use <code class="literal">pub build</code> to generate the final files.
+The [pub tool](/tools/pub) has commands for managing packages,
+managing the system cache, running (or serving) apps,
```

Comment:
"managing the system cache" seems redundant with "managing packages" to me. The notion of a system cache doesn't really make sense outside of a package context anyway.